### ISSUE TITLE
fix: unable to resolve class org.yaml.snakeyaml.Yaml

### ISSUE THREAD

**Issue by SebastianKlaiber** [shorebirdtech/shorebird#1152]
**Description**

After upgrading shorebird to the latest version, I get the following exception if I try to build a patch or release with shorebird. I have already cleaned the cache but get the same result.

```
✓ Fetching apps (0.2s)
✗ Failed to build: Flutter assets will be downloaded from https://download.shorebird.dev. Make sure you trust this source!

FAILURE: Build failed with an exception.

* Where:
Script '/Users/sebastianklaiber/.shorebird/bin/cache/flutter/7b63f1bac9879c2b00f02bc8d404ffc4c7f24ca2/packages/flutter_tools/gradle/src/main/groovy/flutter.groovy' line: 34

* What went wrong:
Could not compile script '/Users/sebastianklaiber/.shorebird/bin/cache/flutter/7b63f1bac9879c2b00f02bc8d404ffc4c7f24ca2/packages/flutter_tools/gradle/src/main/groovy/flutter.groovy'.
> startup failed:
  script '/Users/sebastianklaiber/.shorebird/bin/cache/flutter/7b63f1bac9879c2b00f02bc8d404ffc4c7f24ca2/packages/flutter_tools/gradle/src/main/groovy/flutter.groovy': 34: unable to resolve class org.yaml.snakeyaml.Yaml
   @ line 34, column 1.
     import org.yaml.snakeyaml.Yaml
     ^

  1 error
```

**Steps To Reproduce**

1. shorebird patch android --flavor prod or shorebird release android --flavor prod

**Additional Context**

shorebird doctor
Shorebird 0.14.2 • git@github.com:shorebirdtech/shorebird.git
Flutter 3.13.0 • revision 7b63f1bac9879c2b00f02bc8d404ffc4c7f24ca2
Engine • revision a4613505045177f96d7cbfbb42335e2db7cf7b51

✓ Shorebird is up-to-date (0.5s)
✓ Flutter install is correct (1.2s)
✓ AndroidManifest.xml files contain INTERNET permission (30ms)

No issues detected!

**Comment by eseidel** [comment]
Sorry for the trouble.  We're working on a fix right now, should be out in an hour or so.

**Comment by JunioJsv** [comment]
classpath group : 'org.yaml', name: 'snakeyaml', version: '1.24'

I put this line in the part where I declare the dependencies, in the same file that is giving flutter.groovy error

**Comment by JunioJsv** [comment]
classpath group : 'org.yaml', name: 'snakeyaml', version: '1.24'

I put this line in the part where I declare the dependencies, in the same file that is giving flutter.groovy error, and the error disappeared

**Comment by JunioJsv** [comment]
classpath group : 'org.yaml', name: 'snakeyaml', version: '1.24'

I put this line in the part where I declare the dependencies, in the same file that is giving flutter.groovy error, and the error disappeared

**Comment by eseidel** [comment]
This should be fixed in 0.14.3, just released.  Sorry for the trouble.
